tweb/src/main/java/io/fundreqest/platform/tweb/fund/FundRestController.java

```diff
@@ -18,7 +18,7 @@ public FundRestController(final FundRequestContractsService fundRequestContracts
         this.fundRequestContractsService = fundRequestContractsService;
     }
 
-    @RequestMapping("/allowed-tokens")
+    @RequestMapping("/fund/allowed-tokens")
```

Comment:
frontend changed as well?

Reply:
This is actually a leftover from a rest service I created and removed during development.
The 'fund/' is taken out from the requestMapping at class level and put at method level. So nothing changes for the frontend.